src/services/slack.service.ts

```diff
@@ -0,0 +1,22 @@
+import fetch from 'node-fetch'
+import { logger } from '../utils/logger'
+
+function sendNotification(text: string) {
```

Comment:
suggestion(blocking): The caller of this function does not know or could await that the notification was sent. Consider adding a return value here so that the caller can be sure that someone was notified.